Teapot.podspec

```diff
@@ -0,0 +1,14 @@
+Pod::Spec.new do | spec |
+  spec.name = "Teapot"
+  spec.version = "2.0.0"
+  spec.summary = "A light-weight URLSession wrapper for building simple API clients"
+  spec.homepage = "https://github.com/elland/Teapot"
```

Comment:
Should this be `bakkenbaeck` or are you getting the whole repo transferred to you?

Reply:
No, we can change it back to bakkenbaeck. It was my account so that the pod publish thing would work 🤞